src/main/java/com/example/EnumDemo/mapper/DressMapper.java

```diff
@@ -0,0 +1,43 @@
+package com.example.EnumDemo.mapper;
+
+import com.example.EnumDemo.dto.DressDto;
+import com.example.EnumDemo.dto.showDressDto;
+import com.example.EnumDemo.entity.DressEntity;
+import com.example.EnumDemo.enumC.Size;
+import org.mapstruct.AfterMapping;
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+import org.mapstruct.Named;
+
+import java.util.List;
+
+@Mapper(componentModel = "spring")
+public interface DressMapper {
+    @Mapping(target = "size", source = "dressEntity.size", qualifiedByName = "getDressSizeToLong")
+    DressDto entityToDto(DressEntity dressEntity);
```

Comment:
As discussed previously, it is good to add a mapping for all the attributes.

Reply:
No usage of entityToDto, hence removed from the code